pkg/collectors/cluster_resource_quota.go

```diff
@@ -7,6 +7,7 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/kube-state-metrics/pkg/metric"
+	"strings"
```

Comment:
Could you move this to the top and followed by an empty line?

Reply:
done